src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -142,6 +144,11 @@ public bool PreferencesDebugMode
             }
         }
 
+        /// <summary>
+        /// Returns all installed packages
```

Comment:
just curious, do you think we'll be updating this to display all packages soon, or adding new properties to show packages which are downloaded but not loaded? And what does installed mean anyway 😉 ?

Reply:
I didn't think that much about the comment when I wrote it. Maybe `Returns all known packages` or just `Returns all packages` is better? But I do assume that this property will contain all packages but with more states in the future.